FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.JobManagement/AzureStorageKeyProvider.cs

```diff
@@ -0,0 +1,30 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Synapse.JobManagement
+{
+    public static class AzureStorageKeyProvider
+    {
+        // The table name must conform to rules: https://docs.microsoft.com/en-us/rest/api/storageservices/understanding-the-table-service-data-model#table-names
+        public static string JobInfoTableName(string agentName) => $"{agentName}JobInfo";
+
+        // The queue name must conform to rules: https://docs.microsoft.com/en-us/rest/api/storageservices/naming-queues-and-metadata#queue-names
+        public static string JobMessageQueueName(string agentName) => $"{agentName.ToLower()}-jobqueue";
+
+        public static string JobInfoPartitionKey(byte queueType, long groupId) => $"{queueType:D3}_{groupId:D20}";
```

Comment:
Maybe use the ```QueueType``` as the dataType of input paramater **queueType**?

Reply:
Also for the rest methods in the PR.

Reply:
Per our offline discussion, `QueueType` is defined by the caller,  and the `JobManagement` should know nothing about it, so byte is used here.